src/tree-internal.component.ts

```diff
@@ -83,13 +92,20 @@ export class TreeInternalComponent implements OnInit, OnChanges, OnDestroy {
   public isSelected = false;
   public isRightMenuVisible = false;
   public isLeftMenuVisible = false;
+  public isChecked = false;
```

Comment:
we can have only check on a tree, can't we?

Reply:
I mean the isChecked property - without creating extra field

Reply:
Not sure I'm following, is there another isChecked property?

Reply:
@TMaster I mean - here we have 2 properties: isChecked in the InternalTreeComponent and this.tree.isChecked. Why can't we use just this.tree.isChecked instead of InternalTreeComponent.isChecked?